### PR TITLE
ADCM-2420 Ansible 2.9 as alternative

### PR DESCRIPTION
**Pull request by acmnu**
The commit adds an alternative variant to run playbooks. From now
a bundle developer could set a venv atts on action or on object
to point to a custom venv.

From now we have two venv:

* default - with Ansible 2.8.x
* 2.9 - with Ansible 2.9.x

If venv is ommited in config.yaml, then we have "default" venv.

### CODE REVIEW

File: python/cm/migrations/0081_ansible_venv.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.7 on 2021-12-19 17:23
```
Comment (a-alferov) [inline python/cm/migrations/0081_ansible_venv.py:1]: I think this migration file is not needed

**Comment by acmnu** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @acmnu! Build has been launched!

File: python/cm/migrations/0081_ansible_venv.py (hunk 1 above)
Comment (acmnu) [reply python/cm/migrations/0081_ansible_venv.py]: Migrations has been merged.

Comment (a-alferov) [reply python/cm/migrations/0081_ansible_venv.py]: Yes, I will fix it

**Comment by acmnu** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @acmnu! Build has been launched!

**Review by a-alferov — APPROVED** [review]

**Pull request merged**